channels/serializers.py

```diff
@@ -312,3 +311,26 @@ def create(self, validated_data):
         api = Api(user=self.context['request'].user)
         channel_name = self.context['view'].kwargs['channel_name']
         return api.add_contributor(validated_data['contributor_name'], channel_name)
+
+
+class ModeratorSerializer(serializers.Serializer):
+    """Serializer for Moderators"""
+    moderator_name = WriteableSerializerMethodField()
+
+    def get_moderator_name(self, instance):
+        """Returns the name for the moderator"""
+        return instance.name
+
+    def validate_moderator_name(self, value):
+        """Validates the moderator name"""
+        if not isinstance(value, str):
+            raise ValidationError("contributor name must be a string")
```

Comment:
Can you change this to `moderator name must be a string`? And also modify the next validation error below